src/main_app/jobs_workers/crop_main_files/upload.py

```
@@ -13,10 +13,21 @@
+
+
+def get_file_text(file_name: str) -> str:
+
+    ...
+
+
 def upload_cropped_file(
     cropped_filename: str,
     cropped_path: Path,
     user: Any,
+    wikitext: str = None,
```

Ruff (0.15.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)